pages/dkp/konvoy/2.2/image-builder/index.md

```diff
@@ -9,3 +9,40 @@ menuWeight: 70
 
 Konvoy Image Builder (KIB) is a complete solution for building
 [Cluster API](https://cluster-api.sigs.k8s.io/) compliant images.
+
+This section describes how to use the KIB to create a Cluster API compliant machine images. Machine images contain configuration information and software to create a specific, pre-configured, operating environment. For example, you can create an image of your current computer system settings and software. The machine image can then be replicated and distributed, creating your computer system for other users. The KIB uses variable overrides to specify base image and container images to use in your new machine image.
+
+## Prerequisites
+
+Before you begin, you must ensure your versions of KIB and DKP are compatible:
+
+-  Download the supported version of the KIB bundle from the KIB Version section of the chart below (prefixed with konvoy-image-bundle) for your Operating System.
+### Supported versions
+
+
+| Kubernetes Support  | Version | 
```

Comment:
I don't know that these are the correct Kubernetes versions for DKP 2.2.

Reply:
You know what, I don't either! HA!  Will check with Eng

Reply:
This should already be in the release notes for 2.2

Reply:
I don't really see the point in having the Kubernetes version stuff  on this page at all, honestly.    It's not on the 2.3 page.  Let's just nuke it.

Reply:
Ok, and just will just keep the KIB version matrix.